app/src/main/java/com/mathroule/demoapp/MainActivity.java

```diff
@@ -28,6 +30,22 @@ public void onClick(View view) {
                         .setAction("Action", null).show();
             }
         });
+
+        try {
+            String text = Helper.getRandomButtonLabel();
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/mathroule/demo-app/blob/602938afe0dadcf5133abec0fe37a99e44f2361a/app/src/main/java/com/mathroule/demoapp/MainActivity.java#L35"></span>Variable ‘text’ should be declared final.</td>
    </tr>
  </tbody>
</table>